sdk/test-utils/recorder/src/recorder.ts

```diff
@@ -95,11 +98,27 @@ export class Recorder {
       upstreamUrl.port === testProxyUrl.port &&
       upstreamUrl.protocol === testProxyUrl.protocol;
 
+    if (requestAlreadyRedirected) {
```

Comment:
I see a `!isLiveMode()` check in line 108 but not here, we should only have recorder related logs in record/playback modes.

Perhaps, the if(isLiveMode()) return; can be added at the top instead and remove the live mode checking below?

Reply:
also sounds good